maas-light.md

```diff
@@ -6,9 +6,416 @@ menu: main
 description: A read only API, containing MDS, GBFS and Zone data
 ---
 
-# MDS
+# MDS 2.0
+This section describes the Mobility Data Specification (MDS) provider API, explained in full on the Open Mobility Foundation's GitHub [v2.0](https://github.com/openmobilityfoundation/mobility-data-specification/blob/release-2.0.0/provider/README.md).
 
-This section describes the Mobility Data Specification (MDS) provider API, explained in full on the Open Mobility Foundation's GitHub [v0.4](https://github.com/openmobilityfoundation/mobility-data-specification/tree/0.4.1/provider), [v1.2](https://github.com/openmobilityfoundation/mobility-data-specification/tree/1.2.0/provider).
+## Authentication
+When setting up the partner agreement with VOI, you will receive an email with a UserID and a link to set a Password, this combination will be used to request the client credentials. 
+
+> A token request.
+
+```shell
+$ curl --location https://api.voiapp.io/v1/partner-apis/token \
+     -X POST -u USER_ID:PASSWORD \
+     --form 'grant_type="client_credentials"'
+```
+
+> A token response.
+
+```shell
+{
+    "access_token": "<access token>",
+    "expires_in": 900,
+    "token_type": "bearer"
+}
+```
+
+OAuth 2.0's `client_credentials grant type (outlined in [RFC6749](https://tools.ietf.org/html/rfc6749#section-4.4)) is used as the authentication and authorization scheme.
+This is sometimes called a two-legged OAuth.
+
+### HTTPS request
+
+`POST api.voiapp.io/v1/partner-apis/token`
+
+### Usage
+
+The token is required in the header, using Bearer authentication over HTTPS, on all API calls to MDS and GBFS endpoints.
+
+| Key           | Value               |
+| ------------- | ------------------- |
+| Authorization | Bearer access-token |
+
+## Versioning
+
+The MDS APIs handle requests for specific versions of the specification from the clients.
+Version is communicated through the use of a custom media-type, `application/vnd.mds+json;version=2.0`.
+
+See Open Mobility Foundation's MDS description on versioning [here](https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/general-information.md#versioning).
+
+<aside class="warning">Current MDS 2.0 implementation only supports versions 2.0, for version 1.2 or 0.4 see [here](#mds-12-and-04).</aside>
+
+### Header
+
+| Key    | Value                                |
+| ------ | ------------------------------------ |
+| Accept | application/vnd.mds+json;version=2.0 |
+
+
+## Trips
+
+> A trips request.
+
+```shell
+# v2.0
+$ curl -H "Accept: application/vnd.mds+json;version=2.0"
+  api.voiapp.io/v1/partner-apis/mds/1/trips?end_time=2024-06-13T10
```

Comment:
Maybe you could explain what a trip is very short and then state the url like
A trip is a trip, you can get trips on this path:
/v1/partner-apis/mds/{zone-id}/trips?end_time={time}

example:
your amazing curl :)